src/oc/ocWrapper.ts

```diff
@@ -428,6 +439,16 @@ export class Oc {
         return new CommandText('oc',  `get ${resourceType}`, args);
     }
 
+    private static getRouteURL(
```

Comment:
Agree with @datho7561 - we probably don't need an extra function here. Otherwise works well.

@msivasubramaniaan Did you get an answer on question about what to choose -  `http://` vs. `https://` for a Route URL?

Reply:
No. I didn't get any answers yet.